app/controllers/ops_controller/diagnostics.rb

```diff
@@ -415,11 +415,7 @@ def db_gc_collection
     else
       add_flash(_("Database Garbage Collection successfully initiated"))
     end
-    render :update do |page|
-      page << javascript_prologue
-      page.replace("flash_msg_divdatabase", :partial => "layouts/flash_msg", :locals => {:div_num => "database"})
-      page << "miqSparkle(false);"
-    end
+    render_flash
```

Comment:
@GregP just wondering if this should be render_flash_and_stop_sparkle instead of render_flash so spinner can be stopped after transaction is complete.